src/static/helpers/slasHelper.ts

```diff
@@ -104,6 +105,8 @@ export const generateCodeChallenge = async (
  * @param parameters.redirectURI - the location the client will be returned to after successful login with 3rd party IDP. Must be registered in SLAS.
  * @param parameters.hint? - optional string to hint at a particular IDP. Guest sessions are created by setting this to 'guest'
  * @param parameters.usid? - optional saved SLAS user id to link the new session to a previous session
+ * @param privateClient - flag to indicate if the client is private or not. Defaults to false.
```

Comment:
```suggestion
 * @param privateClient? - flag to indicate if the client is private or not. Defaults to false.
```

For this one and the other ones please